src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -131,6 +131,16 @@ enum xgq_cmd_log_page_type {
 	XGQ_CMD_LOG_BACKUP_FPT		 = 0xc,
 };
 
+/*
+ * XGQ VMR SC status defines
+ */
+typedef enum _xgq_cmd_sc_status
+{
+    XGQ_CMD_SC_PENDING     = 0x0,
```

Comment:
indent issue, usually we use one "TAB" instead of 4 spaces for XRT linux code. this is aligned with linux checkpatch coding style. It is fine for this PR because you can find lots of "exceptions" still today, but better to have the new code clean with linux checkpatch next time.

Reply:
Noted, Thanks